app.rb

```diff
@@ -0,0 +1,40 @@
+require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/database_connection'
+require_relative './db_connect_at_start'
+require_relative "./lib/peep"
+require_relative "./lib/user"
+
+class ChitterApp < Sinatra::Base
+  configure :development do
+    register Sinatra::Reloader
+  end
+  enable :sessions
+
+  get '/' do
+    
+    @peeps = Peep.all
+    @user = User.find(id: session[:user_id]) 
+    puts "Theres a user #{@user} " if @user
```

Comment:
Best to proof read your code so you don't forget to remove print statements like this, which could be a security risk.